internal/budgetadjustments/events/delete.go

```diff
@@ -0,0 +1,88 @@
+package events
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/nobl9/nobl9-go/sdk"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	"github.com/nobl9/sloctl/internal/budgetadjustments/sdkclient"
+	"github.com/nobl9/sloctl/internal/flags"
+)
+
+type DeleteCmd struct {
+	client          *sdk.Client
+	filepath        string
+	dryRun          bool
+	outputFormat    string
+	fieldSeparator  string
+	recordSeparator string
+	out             io.Writer
+	adjustment      string
+}
+
+//go:embed examples/delete_example.sh
+var deleteExample string
+
+func NewDeleteCmd(clientProvider sdkclient.SdkClientProvider) *cobra.Command {
+	deleteCmd := &DeleteCmd{out: os.Stdout}
+
+	cmd := &cobra.Command{
+		Use:     "delete",
+		Short:   "Delete existing events with new values. Values for eventStart and eventEnd are required.",
```

Comment:
I'm not sure I understand this description, can we frame it differently? What does it mean that I "delete existing events with new values"? What new values :P